eppoclient/client.go

```diff
@@ -80,6 +81,76 @@ func (ec *EppoClient) GetJSONAssignment(flagKey string, subjectKey string, subje
 	return variation, err
 }
 
+type BanditResult struct {
+	Variation string
+	Action    *string
+}
+
+func (ec *EppoClient) GetBanditAction(flagKey string, subjectKey string, subjectAttributes ContextAttributes, actions map[string]ContextAttributes, defaultVariation string) BanditResult {
+	// ignoring the error here as we can always proceed with default variation
+	variation, _ := ec.GetStringAssignment(flagKey, subjectKey, subjectAttributes.toGenericAttributes(), defaultVariation)
+
+	bandit, err := ec.configurationStore.getBanditConfiguration(variation)
+	if err != nil {
```

Comment:
maybe this is a normal go thing, but it seems odd to use an error to signify there is no bandit, as opposed to just returning a `nil` bandit.